### ISSUE TITLE
add demo of bevy_gaussian_splatting streaming from a remote server

### ISSUE THREAD

**Issue by cs50victor** [mosure/bevy_gaussian_splatting#87]
putting up a PR, ( hopefully ), soon.

**Comment by cs50victor** [comment]
@mosure use fly.io? currently have new_media working remotely on a fly.io gpu. Wandering if you'll be able to test it.

**Comment by mosure** [comment]
> @mosure use fly.io? currently have new_media working remotely on a fly.io gpu. wondering if you'll be able to test a similar PR here.

I haven't worked with fly.io, it looks very interesting and I don't know of alternatives re. edge container deployments. I've done some initial work with codespaces + docker + bevy_gaussian_splatting, a more robust setup is required for general use.

**Comment by cs50victor** [comment]
true, bevy currently can't work without winit so the docker container needs x11 or wayland. Current solution is supper hacky but seems like the best solution at the moment (https://github.com/cs50victor/newmedia/blob/main/Dockerfile.prod#L38). Do we close this and wait/experiment with a more robust setup or just go ahead?
